package.json

```diff
@@ -98,7 +98,8 @@
     "react-native-toast-message": "^2.1.7",
     "react-native-url-polyfill": "^2.0.0",
     "react-native-web": "~0.19.9",
-    "react-native-webview": "13.2.2"
+    "react-native-webview": "13.2.2",
+    "use-memo-one": "^1.1.3"
```

Comment:
I am not sure what benefit you get form using this package. It looks like it tries to solve an issue with brute force basically yelling at react to make it do what you want.

Reply:
I removed the package, I used it because the chat library was using it, but you are right and we can use `useCallback` instead.